### PR TITLE
Rehabilitate extraversion

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Extr**a**version is commonly used in scholar articles.

One could argue that extr**o**version is mostly American and extr**a**version mostly British, but it's not as simple as that, since extr**a**version is commonly used in American scholar articles.

For a more detailed explanation, see for example [The Difference between ExtrAversion and ExtrOversion](https://blogs.scientificamerican.com/beautiful-minds/the-difference-between-extraversion-and-extroversion/).

Dictionary entries:
* [APA Dictionary of Psychology](https://dictionary.apa.org/extraversion)
* [Oxford Learner's Dictionary](https://www.oxfordlearnersdictionaries.com/definition/english/extraversion)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/extraversion)
* [Collins](https://www.collinsdictionary.com/dictionary/english/extraversion)
* [Macmillan](https://www.macmillandictionary.com/dictionary/british/extraversion)

### CODE REVIEW

**Comment by skangas** [comment]
> For a more detailed explanation, see for example [The Difference between ExtrAversion and ExtrOversion](https://blogs.scientificamerican.com/beautiful-minds/the-difference-between-extraversion-and-extroversion/).

That was an interesting read. Quoting from the above linked article:

> According to the Oxford English Dictionary, "The original spelling 'Extravert' is now rare in general use but is found in technical use in psychology." That's correct. If you look at scientific journal articles, virtually every paper uses the spelling ExtrAvert.

Since the spelling is commonly in use, I agree that the least opinionated thing here would be to remove this correction. So this LGTM.

**Comment by DimitriPapadopoulos** [comment]
@skangas Could you approve formally now that you can?

**Review by skangas — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@skangas Thank you for re-approving, yet it looks like you're not in the Approvers list after all.

@larsoner This repository probably requires approving reviews from people with write or admin permissions. The alternative might be to add someone to [CODEOWNERS](https://github.com/codespell-project/codespell/blob/master/.github/CODEOWNERS).

[About protected branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-pull-request-reviews-before-merging) is not entirely clear about that:
> You can require approving reviews from people with write permissions in the repository or from a designated code owner.
> 
> If you enable required reviews, collaborators can only push changes to a protected branch via a pull request that is approved by the required number of reviewers with write permissions.

**Comment by larsoner** [comment]
Thanks @DimitriPapadopoulos

**Pull request merged**
